bundle/src/main/java/io/wcm/caconfig/editor/package-info.java

```diff
@@ -20,5 +20,5 @@
 /**
  * Configuration editor API.
  */
-@org.osgi.annotation.versioning.Version("1.5")
+@org.osgi.annotation.versioning.Version("2.0")
```

Comment:
we must stay at 1.x for backward compatibility

Reply:
I changed it back to 1.6 after refactoring the RootPathProvider to 2 separate implementations